opendcs-rest-api/src/main/java/org/opendcs/odcsapi/res/ObjectMapperContextResolver.java

```diff
@@ -41,11 +46,13 @@ public ObjectMapper getContext(Class<?> type)
 
     private ObjectMapper createObjectMapper() 
     {
-    	System.out.println("createObjectMapper");
-        ObjectMapper mapper = new ObjectMapper();
-        SimpleDateFormat sdf = new SimpleDateFormat("yyyy-MM-dd'T'HH:mm:ss.SSS'Z'[z]");
-        sdf.setTimeZone(TimeZone.getTimeZone("UTC"));
-        mapper.setDateFormat(sdf);
-        return mapper;
+        String dateFormat = "yyyy-MM-dd'T'HH:mm:ss.SSS'Z'[z]";
+        String timezone = "UTC";
+        LOGGER.info("Creating object map using date format '{}' and timezone '{}'", dateFormat, timezone);
```

Comment:
I am not sure what detail of logging is desired here.  I didn't fully think the system.out was necessary, but I do want to add some logging here.  Let me know if this type of logging is desired and I will keep it.  Otherwise, I can remove it.

Not sure if logging the dateformat and timezone is necessary either.  I can remove it if it's too much.

Reply:
Honestly probably nothing unless there's an error. That particular line of code is going to get executed so often (basically for everything) that it would be difficult for all of us to not know it's correct.

Reply:
Sounds good.  In that case I will just remove the system.out.